[2020-05-26 00:54:38,832][MainThread:4625432000][task_id:django.utils.autoreload][autoreload.py:598][INFO][Watching for file changes with StatReloader]
[2020-05-26 00:54:39,045][django-main-thread:123145449775104][task_id:apscheduler.scheduler][base.py:166][INFO][Scheduler started]
[2020-05-26 00:54:39,139][django-main-thread:123145449775104][task_id:daphne.server][server.py:111][INFO][HTTP/2 support not enabled (install the http2 and tls Twisted extras)]
[2020-05-26 00:54:39,140][django-main-thread:123145449775104][task_id:daphne.server][server.py:119][INFO][Configuring endpoint tcp:port=8000:interface=127.0.0.1]
[2020-05-26 00:54:39,141][django-main-thread:123145449775104][task_id:daphne.server][server.py:153][INFO][Listening on TCP address 127.0.0.1:8000]
[2020-05-26 00:54:45,894][django-main-thread:123145449775104][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/power_on 200 [0.01, 127.0.0.1:50867][0m]
[2020-05-26 00:54:45,917][django-main-thread:123145449775104][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/change_state 200 [0.01, 127.0.0.1:50867][0m]
[2020-05-26 00:54:50,102][django-main-thread:123145449775104][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.03, 127.0.0.1:50867][0m]
[2020-05-26 00:54:52,673][django-main-thread:123145449775104][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.01, 127.0.0.1:50867][0m]
[2020-05-26 00:54:56,081][django-main-thread:123145449775104][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.02, 127.0.0.1:50867][0m]
[2020-05-26 00:54:58,630][django-main-thread:123145449775104][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.02, 127.0.0.1:50867][0m]
[2020-05-26 00:55:01,726][django-main-thread:123145449775104][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.02, 127.0.0.1:50867][0m]
[2020-05-26 00:55:05,007][django-main-thread:123145449775104][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.01, 127.0.0.1:50867][0m]
[2020-05-26 00:55:07,689][django-main-thread:123145449775104][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.01, 127.0.0.1:50867][0m]
[2020-05-26 00:57:29,725][MainThread:4783902144][task_id:django.utils.autoreload][autoreload.py:598][INFO][Watching for file changes with StatReloader]
[2020-05-26 00:57:29,826][django-main-thread:123145522884608][task_id:apscheduler.scheduler][base.py:166][INFO][Scheduler started]
[2020-05-26 00:57:29,894][django-main-thread:123145522884608][task_id:daphne.server][server.py:111][INFO][HTTP/2 support not enabled (install the http2 and tls Twisted extras)]
[2020-05-26 00:57:29,895][django-main-thread:123145522884608][task_id:daphne.server][server.py:119][INFO][Configuring endpoint tcp:port=8000:interface=127.0.0.1]
[2020-05-26 00:57:29,896][django-main-thread:123145522884608][task_id:daphne.server][server.py:153][INFO][Listening on TCP address 127.0.0.1:8000]
[2020-05-26 00:57:29,935][<concurrent.futures.thread.ThreadPoolExecutor object at 0x1131e23c8>_0:123145533394944][task_id:django.request][log.py:228][ERROR][Internal Server Error: /customer/poll]
Traceback (most recent call last):
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/Users/wmc/vscode/django/django-air/customer/views.py", line 172, in poll
    pause(room_id, phone_num)
  File "/Users/wmc/vscode/django/django-air/master/mainMachine.py", line 318, in pause
    machine.new_request(room_id=s.room_id, phone_num=phone_num, req_time=int(time.time()), sp_mode=s.sp_mode)
AttributeError: 'NoneType' object has no attribute 'room_id'
[2020-05-26 00:57:29,938][django-main-thread:123145522884608][task_id:django.channels.server][runserver.py:154][ERROR][[35;1mHTTP POST /customer/poll 500 [0.03, 127.0.0.1:51636][0m]
[2020-05-26 00:57:31,973][<concurrent.futures.thread.ThreadPoolExecutor object at 0x1131e23c8>_1:123145538650112][task_id:django.request][log.py:228][ERROR][Internal Server Error: /customer/poll]
Traceback (most recent call last):
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/Users/wmc/vscode/django/django-air/customer/views.py", line 172, in poll
    pause(room_id, phone_num)
  File "/Users/wmc/vscode/django/django-air/master/mainMachine.py", line 318, in pause
    machine.new_request(room_id=s.room_id, phone_num=phone_num, req_time=int(time.time()), sp_mode=s.sp_mode)
AttributeError: 'NoneType' object has no attribute 'room_id'
[2020-05-26 00:57:31,978][django-main-thread:123145522884608][task_id:django.channels.server][runserver.py:154][ERROR][[35;1mHTTP POST /customer/poll 500 [0.04, 127.0.0.1:51636][0m]
[2020-05-26 00:57:32,029][<concurrent.futures.thread.ThreadPoolExecutor object at 0x1131e23c8>_0:123145533394944][task_id:django.request][log.py:228][ERROR][Internal Server Error: /customer/poll]
Traceback (most recent call last):
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/Users/wmc/vscode/django/django-air/customer/views.py", line 172, in poll
    pause(room_id, phone_num)
  File "/Users/wmc/vscode/django/django-air/master/mainMachine.py", line 318, in pause
    machine.new_request(room_id=s.room_id, phone_num=phone_num, req_time=int(time.time()), sp_mode=s.sp_mode)
AttributeError: 'NoneType' object has no attribute 'room_id'
[2020-05-26 00:57:32,036][django-main-thread:123145522884608][task_id:django.channels.server][runserver.py:154][ERROR][[35;1mHTTP POST /customer/poll 500 [0.04, 127.0.0.1:51636][0m]
[2020-05-26 00:57:34,832][<concurrent.futures.thread.ThreadPoolExecutor object at 0x1131e23c8>_1:123145538650112][task_id:django.request][log.py:228][ERROR][Internal Server Error: /customer/poll]
Traceback (most recent call last):
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/Users/wmc/vscode/django/django-air/customer/views.py", line 172, in poll
    pause(room_id, phone_num)
  File "/Users/wmc/vscode/django/django-air/master/mainMachine.py", line 318, in pause
    machine.new_request(room_id=s.room_id, phone_num=phone_num, req_time=int(time.time()), sp_mode=s.sp_mode)
AttributeError: 'NoneType' object has no attribute 'room_id'
[2020-05-26 00:57:34,835][django-main-thread:123145522884608][task_id:django.channels.server][runserver.py:154][ERROR][[35;1mHTTP POST /customer/poll 500 [0.02, 127.0.0.1:51636][0m]
[2020-05-26 00:57:34,856][<concurrent.futures.thread.ThreadPoolExecutor object at 0x1131e23c8>_2:123145543905280][task_id:django.request][log.py:228][ERROR][Internal Server Error: /customer/poll]
Traceback (most recent call last):
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wmc/opt/anaconda3/envs/django/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/Users/wmc/vscode/django/django-air/customer/views.py", line 172, in poll
    pause(room_id, phone_num)
  File "/Users/wmc/vscode/django/django-air/master/mainMachine.py", line 318, in pause
    machine.new_request(room_id=s.room_id, phone_num=phone_num, req_time=int(time.time()), sp_mode=s.sp_mode)
AttributeError: 'NoneType' object has no attribute 'room_id'
[2020-05-26 00:57:34,861][django-main-thread:123145522884608][task_id:django.channels.server][runserver.py:154][ERROR][[35;1mHTTP POST /customer/poll 500 [0.02, 127.0.0.1:51636][0m]
[2020-05-26 00:57:47,148][MainThread:4641942976][task_id:django.utils.autoreload][autoreload.py:598][INFO][Watching for file changes with StatReloader]
[2020-05-26 00:57:47,243][MainThread:4641942976][task_id:apscheduler.scheduler][base.py:166][INFO][Scheduler started]
[2020-05-26 00:57:47,310][django-main-thread:123145465171968][task_id:daphne.server][server.py:111][INFO][HTTP/2 support not enabled (install the http2 and tls Twisted extras)]
[2020-05-26 00:57:47,310][django-main-thread:123145465171968][task_id:daphne.server][server.py:119][INFO][Configuring endpoint tcp:port=8000:interface=127.0.0.1]
[2020-05-26 00:57:47,311][django-main-thread:123145465171968][task_id:daphne.server][server.py:153][INFO][Listening on TCP address 127.0.0.1:8000]
[2020-05-26 00:57:51,338][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP GET /customer/ 200 [0.01, 127.0.0.1:51741][0m]
[2020-05-26 00:57:51,564][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP GET /static/customer/action_copy.js 200 [0.01, 127.0.0.1:51741][0m]
[2020-05-26 00:57:51,565][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP GET /static/css/fonts.css 200 [0.01, 127.0.0.1:51742][0m]
[2020-05-26 00:57:51,566][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP GET /static/js/bootstrap.min.js 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:57:51,567][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP GET /static/css/bootstrap.min.css 200 [0.01, 127.0.0.1:51743][0m]
[2020-05-26 00:57:51,719][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP GET /static/images/snow.png 200 [0.01, 127.0.0.1:51743][0m]
[2020-05-26 00:57:51,726][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP GET /static/images/speed.png 200 [0.01, 127.0.0.1:51743][0m]
[2020-05-26 00:57:51,768][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP GET /static/fonts/glyphicons-halflings-regular.woff2 200 [0.01, 127.0.0.1:51743][0m]
[2020-05-26 00:57:51,781][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP GET /static/js/bootstrap.min.js.map 200 [0.01, 127.0.0.1:51743][0m]
[2020-05-26 00:57:51,790][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP GET /static/css/bootstrap.min.css.map 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:57:51,912][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP GET /customer/get_default 200 [0.00, 127.0.0.1:51744][0m]
[2020-05-26 00:57:51,933][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:147][INFO][[32mHTTP GET /favicon.ico 301 [0.00, 127.0.0.1:51744][0m]
[2020-05-26 00:57:51,941][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP GET /static/images/favicon.ico 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:57:57,560][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/power_on 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:57:57,581][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/change_state 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:58:00,660][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:58:03,631][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:58:06,892][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:58:09,723][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:58:12,641][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:58:15,954][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:58:18,704][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:58:21,586][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/poll 200 [0.01, 127.0.0.1:51744][0m]
[2020-05-26 00:58:22,038][django-main-thread:123145465171968][task_id:django.channels.server][runserver.py:141][INFO][[mHTTP POST /customer/power_off 200 [0.01, 127.0.0.1:51744][0m]
